.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule ".build/tangle"]
 	path = .build/tangle
 	url = https://github.com/ehildenb/pandoc-tangle
+[submodule "tests/ci/kore"]
```

Comment:
Can we move both of these to .build?

Reply:
You mean moving both `kore` and `rv-k`, right?

Reply:
Done. Those submodules are now in `.build/rv-k` and `.build/kore`. 
 
What was previously `.build/kore/iele-testing.kore` is now `.build/llvm/iele-testing.kore`, and the `Makefile` target has been renamed from `kore` to `llvm` accordingly.